src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/util/Writer.java

```diff
@@ -71,12 +71,14 @@ public void doHttpRequest(
         MessageConsumer receiver,
         long routeId,
         long streamId,
-        Consumer<ListFW.Builder<HttpHeaderFW.Builder, HttpHeaderFW>> mutator)
+        Consumer<ListFW.Builder<HttpHeaderFW.Builder, HttpHeaderFW>> mutator,
+        long traceId)
     {
         final BeginFW begin = beginRW.wrap(writeBuffer, 0, writeBuffer.capacity())
                 .routeId(routeId)
                 .streamId(streamId)
                 .extension(e -> e.set(visitHttpBeginEx(mutator)))
+                .trace(traceId)
```

Comment:
IIRC, this will fail at runtime unless `trace(traceId)` is immediately after `streamId(streamId)`.